### PR TITLE
feat: firefox-stable channel

### PR DESCRIPTION
**Pull request by kumaraditya303**
https://github.com/microsoft/playwright/commit/653d483ba87190a9fa727fe71c40be202ce9132f

### CODE REVIEW

File: playwright/async_api/_generated.py [hunk 1]
```diff
@@ -9185,7 +9186,7 @@ async def launch(
             Path to a browser executable to run instead of the bundled one. If `executablePath` is a relative path, then it is
             resolved relative to the current working directory. Note that Playwright only works with the bundled Chromium, Firefox
             or WebKit, use at your own risk.
-        channel : Union["chrome", "chrome-beta", "chrome-canary", "chrome-dev", "msedge", "msedge-beta", "msedge-canary", "msedge-dev", NoneType]
+        channel : Union["chrome", "chrome-beta", "chrome-canary", "chrome-dev", "msedge", "msedge-beta", "msedge-canary", "msedge-dev", "firefox-stable", NoneType]
```
Comment (kumaraditya303) [inline playwright/async_api/_generated.py:9189]: Why linting is failing no idea as the api is generated yet it is failing

**Review by kumaraditya303 — COMMENTED** [review]
Fixed tests

**Review by pavelfeldman — COMMENTED** [review]
I don't think we need the stable channels bots downstream, upstream should be enough to cover it.

**Comment by pavelfeldman** [comment]
This looks red all over!

**Comment by kumaraditya303** [comment]
@pavelfeldman Failing tests are because docs are not added to upstream yet

**Review by mxschmitt — APPROVED** [review]

File: tests/async/test_navigation.py [hunk 2]
```diff
@@ -969,7 +969,7 @@ def expect_ssl_error(error_message: str, browser_name: str) -> None:
         if sys.platform == "darwin":
             assert "The certificate for this server is invalid" in error_message
         elif sys.platform == "win32":
-            assert "SSL connect error" in error_message
+            assert "SSL peer certificate" in error_message
```
Comment (mxschmitt) [inline tests/async/test_navigation.py:972]:
```suggestion
            assert "SSL peer certificate or SSH remote key was not OK" in error_message
```

**Pull request merged**